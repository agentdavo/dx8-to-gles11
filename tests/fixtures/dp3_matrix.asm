dp4 oPos, c0, v0
dp3 r0, v0, v1
