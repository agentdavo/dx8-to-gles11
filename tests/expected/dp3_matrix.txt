MATRIX_MODE mode=5888
TEX_ENV_COMBINE func=34478
